src/DiscordMember.js

```diff
@@ -227,6 +227,21 @@ class DiscordMember {
         }
       }
 
+      // See if the server currently uses nicknames.
```

Comment:
- I would move this logic to DiscordMember.getNickname and remove the if statement here.
- You can't store the group rank in the user data directly because that mutates the `data` object and that is cached and shared across all servers. In DiscordMember.getNickname, construct a new object with the keys `robloxId`, `robloxUsername`, and `groupRank` before send it off to Util.formatDataString.
- If the server has no nicknameGroup, you need to send an empty string for `groupRank`.